webapi/Controllers/PluginController.cs

```diff
@@ -48,9 +48,9 @@ public PluginController(
     /// <param name="manifestDomain">The domain of the manifest.</param>
     /// <returns>The plugin's manifest JSON.</returns>
     [HttpGet]
-    [Route("pluginManifests")]
+    [Route("pluginManifests")] // TODO: Fix name and test
```

Comment:
Question: is the TODO intended?

Reply:
yes, I believe the name should be singular